pkg/controller/workload/job/job_controller.go

```diff
@@ -50,6 +53,10 @@ var (
 	parentWorkloadKey = ".metadata.parentWorkload"
 )
 
+const (
+	sourceSpecHashKey = "kueue.x-k8s.io/source-hash"
```

Comment:
worth putting this in a more visible place, like `pkg/constants`

Reply:
done